specs/0119-transactional-emails.md

```diff
@@ -0,0 +1,428 @@
+
+Finally, some additional transactional emails are intended to complement upcoming initiatives for improving the onboarding experience for new contributors.
+
+# Feature explanation
+
+## Settings changes
+
+### Sending comments as email
+
+Users choose what notifications are sent to their email by an “Include in email digest” setting added under the existing “Notification subscriptions” section in settings. No email digest is sent if there are no notifications selected, otherwise an email digest is sent either daily or weekly based on the user’s selection from an “Email digest frequency” dropdown.
+
+This updates the “Notification subscriptions” section to include:
+
+* 2 columns with checkboxes for each notification type.  
+* Column 1 label: “Enable”  
+  * This maps to the current functionality — enabling this will enable notifications in Pontoon.  
+  * Specs / defaults should be preserved from current implementation.  
+* Column 2 label (default value: disabled): “Include in email digest”  
+  * When enabled, this type of notification will be included in an email sent to the user’s designated email address (either their login account, or the email address they define in the email preferences section.)  
+* Dropdown list (default value: Daily): “Email digest frequency”  
+  * Options: “Daily”, “Weekly”  
+  * Disable list if nothing enabled in the “Include in email digest” column.
+
+![](./0119/subscription-settings.png)
+
+### Monthly activity summary
+
+Users enable or disable a monthly activity report with the “Monthly activity summary” option located in the Email section of settings. This sets an enabled/disabled flag for the user that is referenced to determine if an automated email should be sent to this user.
+
+* Header: “Monthly activity summary”  
+  * Located under “Email” section  
+  * Contributors are opted-out by default, translators/team managers are opted-in by default in order to reduce the amount of emails being sent.  
+* Checkbox (default: see below):  “Receive monthly activity summary email”  
+  * Each user model has a flag that corresponds to this checkbox. The flag is referenced by the transactional email trigger to determine if an email is sent. This flag is disabled by default for users except translators and managers.  
+  * Flag is enabled for all current users with global translator or manager permissions at time of implementation.  
+  * Flag is automatically enabled for a user who receives new global translator or manager permissions. If a user's translator or manager permissions are removed, the user would have to manually disable these emails from their settings.
+
+## Transactional emails
```

Comment:
```suggestion
## New transactional emails
```

(to make a distinction against the following section called "Existing Transactional Emails")